tools/import_place_from_production.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+if Rails.env.production?
```

Comment:
`unless Rails.env.development?` ?

Reply:
Fair. No reason to be running this in a test environment.

Reply:
Sure, this was just copy-pasted from another file in the directory.

Reply:
Addressed in [bc29c2f](https://github.com/inaturalist/inaturalist/pull/3655/commits/bc29c2fe096d0d50047db774aa00685014d4cf20).